WaterLog_Backend/Controllers/SegmentEventsController.cs

```diff
@@ -190,290 +197,55 @@ public async Task<List<GetAlerts>> GetAlertsByPage(int id = 1)
-                                    (entry
-                                    .AttendedDate
-                                    .Subtract(entry.AttendedDate) < TimeSpan.Zero ? TimeSpan.Zero : entry
-                                    .FaultDate.
-                                    Subtract(entry
-                                    .AttendedDate)),
-                                    Globals.Sensor,
-                                    entry.SensorId,
-                                    Globals.Faulty,
-                                   -1,
-                                   -1,
-                                   "High",
-                                   latestReading.PercentageLevel,
-                                   -1,
-                                   entry.SensorResolved
-                                )
-                            );
-                        }
-                    }
-                    else
-                    {
-                        var sensorInfo = await _db.Monitors.Where(a => a.Id == entry.SensorId).FirstOrDefaultAsync();
-                        var latestReading = await _db.Readings
-                            .Where(a => a.MonitorsId == entry.SensorId)
-                            .OrderByDescending(a => a.TimesStamp).FirstOrDefaultAsync();
-                        if (entry.SensorId == sensorId && (int)entry.SensorType == (int)EnumSensorType.WATER_FLOW_SENSOR)
-                        {
-                            ListOfAlerts.Add(
-                                new GetAlerts
-                                (
-                                    entry.FaultDate,
-                                     (entry
-                                    .AttendedDate
-                                    .Subtract(entry.AttendedDate) < TimeSpan.Zero ? TimeSpan.Zero : entry
-                                    .FaultDate.
-                                    Subtract(entry
-                                    .AttendedDate)),
-                                    Globals.Sensor,
-                                    entry.SensorId,
-                                    Globals.Sensor,
-                                    0.0,
-                                    0.0,
-                                    "High",
-                                    latestReading.Value,
-                                    sensorInfo.Max_flow,
-                                    entry.SensorResolved
-                                )
-                            );
-                        }
-                    }
-                }
-            } // filter faulty sensor end
-            return (ListOfAlerts.OrderByDescending(a => a.Date).OrderByDescending(a => a.Status).ToList());
+                return alerts
+                     .Where(alert => alert.Severity.ToLower() == filter.Severity.ToLower())
+                     .ToList();
+            } else if (filteredAlerts.Count != 0)
```

Comment:
put elseIf on the second line to bring consistency

Reply:
brought consistency